### ISSUE TITLE
Issues with latest pofile

### ISSUE THREAD

**Issue by jwalton** [connectedcars/gettext-extractor#32]
I just deleted my package-lock.json and reinstalled everything, so I'm pickup up the latest `gettext-extractor` and `pofile`, and I'm getting these errors in my build:

```
node_modules/gettext-extractor/dist/extractor.d.ts(1,25): error TS2497: This module can only be referenced with ECMAScript imports/exports by turning on the 'allowSyntheticDefaultImports' flag and referencing its default export.
node_modules/gettext-extractor/dist/extractor.d.ts(23,36): error TS2702: 'pofile' only refers to a type, but is being used as a namespace here.
node_modules/gettext-extractor/dist/extractor.d.ts(24,53): error TS2702: 'pofile' only refers to a type, but is being used as a namespace here.
node_modules/gettext-extractor/dist/extractor.d.ts(25,58): error TS2702: 'pofile' only refers to a type, but is being used as a namespace here.
```

**Comment by jwalton** [comment]
Yeah, forcing `pofile` down to 1.0.11 via package-lock.json seems to fix this.

**Comment by lukasgeiter** [comment]
Thanks a lot for reporting this!

`pofile` completely rewrote the types in the latest release causing this issue. Unfortunately I was not able to make my code work with the new types ([and I don't seem to be the only one](https://github.com/rubenv/pofile/issues/36)) so instead I've locked the version to `1.0.x`. The fix is released with version 3.5.1.

**Comment by jwalton** [comment]
Thanks for the quick fix!  :)
